README.md

```diff
@@ -40,6 +40,12 @@ yarn
 yarn build
 ```
 
+### Using local core contracts
```

Comment:
Maybe include a word/line that this is optional? It being in setup section makes it seem mandatory